pkg/virtualization/core/service/gpu.go

```diff
@@ -43,7 +43,7 @@ func (vmms *VirtualSystemManagementService) AttachGPU(vm *virtualsystem.VirtualM
 	return
 }
 
-func (vmms *VirtualSystemManagementService) DetachGPU(vm *virtualsystem.VirtualMachine, hostResource string) (err error) {
+func (vmms *VirtualSystemManagementService) DetachGpuDda(vm *virtualsystem.VirtualMachine, hostResource string) (err error) {
 	gpu, err := vm.GetPcieDevice(hostResource)
 	if err != nil {
 		log.Printf("[WMI] Error getting GPU device for host resource [%s] - Error details [%+v]\n", hostResource, err)
```

Comment:
Why dont we return this kind of error back, instead of a blank nonfound

Reply:
Removed logs statements and returning detailed error message.